data/benefits/aides-velo-generator.js

```diff
@@ -11,30 +11,32 @@ function generate_benefit_list(institutions) {
   }
 
   benefits.forEach((b) => {
-    switch (b?.collectivity?.kind) {
-      case "pays": {
-        if (b.collectivity.value === "France") {
-          b.institution = "etat"
-        } else {
-          b.discard = true
+    if (b && b.collectivity) {
```

Comment:
Peut-on imposer dans cette partie du code cette contrainte là via TypeScript ?

Reply:
Bon en fait, c'est un fichier JS. :sweat: 
```suggestion
    if (b && !b.collectivity) {
        throw new Error("Ouahhh")
    } else {
```

Reply:
J'ai essayé d'aller dans votre sens, qu'en pensez-vous ?